### PR TITLE
genpass: init at 0.4.1

### PR DESCRIPTION
**Pull request by cyplo**
- [X] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [X] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/security/genpass/default.nix [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+, pkgconfig
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "genpass";
+  version = "0.4.1";
+
+  src = fetchFromGitHub {
+    owner = "cyplo";
+    repo = pname;
+    rev = "v" + version;
```
Comment (cole-h) [inline pkgs/tools/security/genpass/default.nix:15]:
```suggestion
    rev = "v${version}";
```

File: pkgs/tools/security/genpass/default.nix [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    sha256 = "1b22m7g55k5ry0vwyd8pakh8rmfkhk37qy5r74cn3n5pv3fcwini";
+  };
+
+  cargoSha256 = "1p6l64s9smhwka8bh3pamqimamxziad859i62nrmxzqc49nq5s7m";
+
+  buildInputs = [
+    git openssl pkgconfig
```
Comment (cole-h) [inline pkgs/tools/security/genpass/default.nix:22]:
```suggestion
    git
    openssl
    pkgconfig
```

(just looks nicer)

File: maintainers/maintainer-list.nix [hunk 3]
```diff
@@ -1646,6 +1646,12 @@
     githubId = 1382175;
     name = "Oliver Matthews";
   };
+  cyplo = {
```
Comment (cole-h) [inline maintainers/maintainer-list.nix:1649]: I'd like to humbly request that you split this out to a separate commit, please: `maintainers: add cyplo`

Also, you should be below `cypherpunk2140` -- `h` comes before `l` ;)

**Review by cole-h — CHANGES_REQUESTED** [review]

File: maintainers/maintainer-list.nix (hunk 3 above)
Comment (cyplo) [reply maintainers/maintainer-list.nix]: must have been late :D

**Comment by cyplo** [comment]
thanks so much for a comprehensive review @cole-h ! 
all should be changed now - if you could take a look ? :)

File: pkgs/tools/security/genpass/default.nix [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+  };
+
+  cargoSha256 = "1p6l64s9smhwka8bh3pamqimamxziad859i62nrmxzqc49nq5s7m";
+
+  buildInputs = [
+    git
+    openssl
+    pkgconfig
+  ];
+
```
Comment (cole-h) [inline pkgs/tools/security/genpass/default.nix:26]:
```suggestion
```

I was able to build without these. (Also should be removed from the top of the file.)

Comment (cyplo) [reply pkgs/tools/security/genpass/default.nix]: ah yes, good catch !

**Review by cole-h — APPROVED** [review]
Diff LGTM. Binary displays help and does indeed generate passwords/phrases.

```
[1 built, 0.0 MiB DL]
https://github.com/NixOS/nixpkgs/pull/86504
1 package built:
genpass
```

**Comment by cyplo** [comment]
heya @cole-h - would you know who bes tto ping here to get this one merged ? thanks ! :)

**Review by bjornfor — APPROVED** [review]
